compute/README.md

```diff
@@ -0,0 +1,55 @@
+---
+page_type: sample
+languages:
+- golang
+products:
+- azure
+description: "These code samples will show you how to manage Compute using Azure SDK for Golang."
```

Comment:
Is it better to use 'Azure Compute service'?

Reply:
reference https://github.com/Azure-Samples/azure-samples-js-management